presto-docs/src/main/sphinx/admin/function-namespace-managers.rst

```diff
@@ -0,0 +1,50 @@
+===========================
+Function Namespace Managers
+===========================
+
+.. warning::
+
+    This is an experimental feature being actively developed. The way
+    Function Namespace Managers are configured might be changed.
+
+Function namespace managers support storing and retrieving user-defined
+functions with a persistent storage, allowing users to create, alter,
+and delete functions, as well as to reference them within queries.
+
+Presto provides a MySQL-based function namespace manager. It is
+pluggable and can be configured by an administrator.
+
+To instantiate a MySQL-based function namespace manager that manages
+catalog ``example``, administrator needs to first have a running MySQL
+server. Suppose the MySQL server can be reached at ``localhost:1080``,
+add a file ``etc/function-namespace/example.properties`` with the
+following contents::
+
+    function-namespace-manager.name=mysql
+    database_url=localhost:1080
+
+When Presto first starts with the above MySQL function namespace
+manager configuration, two MySQL tables are created.
+
+- ``function_namespaces``: Stores all function namespaces.
```

Comment:
Use the configured table name from the example config instead.

The concept of "function namespace" was never officially introduced and defined. Please give a better explanation. What does "all function namespaces" mean? Maybe "allowed function namespaces"?